mllib/src/test/scala/org/apache/spark/mllib/tree/RandomForestSuite.scala

```diff
@@ -196,6 +196,24 @@ class RandomForestSuite extends FunSuite with MLlibTestSparkContext {
       featureSubsetStrategy = "sqrt", seed = 12345)
     EnsembleTestHelper.validateClassifier(model, arr, 1.0)
   }
+
+  test("subsampling rate in RandomForest"){
+    val arr = EnsembleTestHelper.generateOrderedLabeledPoints(5, 20)
+    val rdd = sc.parallelize(arr)
+    val strategy1 = new Strategy(algo = Classification, impurity = Gini, maxDepth = 2,
```

Comment:
I would make 1 instance of strategy, train rf1, modify the strategy's subsamplingRate, and train rf2.  Simpler + more clearly using the same settings for other parameters